tests/test_models.py

```diff
@@ -475,3 +476,52 @@ def test_sse_event_handler(simai_client, model_factory):
         }
     )
     assert model.is_ready
+
+
+@responses.activate
+def test_post_process_input(simai_client):
+    """WHEN ModelConfiguration includes a specified surface_pp_input arg
+    THEN Model object configuration includes that exact surface_pp_input
+    """
+
+    raw_project = {
+        "id": MODEL_RAW["project_id"],
+        "name": "fifi",
+        "sample": SAMPLE_RAW,
+    }
+
+    responses.add(
+        responses.GET,
+        f"https://test.test/projects/{MODEL_RAW['project_id']}",
+        json=raw_project,
+        status=200,
+    )
+
+    project: Project = simai_client._project_directory._model_from(raw_project)
+    project.verify_gc_formula = Mock()
+
+    MODEL_CONFIG = MODEL_CONF_RAW.copy()
```

Comment:
uppercase is generally used for global constants, this looks like a local variable